src/app/components/navbar/navbar.component.scss

```diff
@@ -0,0 +1,84 @@
+@use '~@angular/material' as material;
+@import '../../theme/app.theme.scss';
+
+#navbar {
+    position: fixed;
+    left: 0px;
+    top: 0px;
+    right: 0px;
+    height: 75px;
+    min-height: 20px;
+    display: flex;
+    flex-direction: row;
+    padding: 10px 15px 0px 15px;
+    background: rgba(material.get-color-from-palette($blue-palette, 500), 0.85)
+}
+
+img {
+    height: 50%;
+    width: auto;
+    margin-right: 10px;
+}
+
+#navbar-title {
+    position: relative;
+    display:flex;
+    align-items: center;
+    align-self: flex-start;
+    height: 100%;
+}
+
+#navbar-items {
+    position: relative;
+    display: flex;
+    flex-direction: row;
+    flex-grow:8;
+    align-items: center;
+    justify-content: center;
+    align-self: center;
+    height: 75%;
+}
+
+#navbar-item {
+    display: flex;
+    align-self: center;
+    margin-left: 15px;
+    margin-right: 15px;
+    height: auto;
+    box-sizing: border-box;
+    text-align: center;
+    border-bottom: 3px solid transparent;
+    transition: all .5s ease;
+    backface-visibility: hidden;
+}
+
+p {
+    backface-visibility: hidden;
+    line-height: 16px;
+}
+
+#navbar-item:hover p{
+    color: material.get-color-from-palette($pink-palette, 500);
+    transition: all .5s ease;
+}
+
+#navbar-item:hover{
+    transform: perspective(1px) scale(1.1);
+    transition: all .5s ease;
+}
+
+
+div.isSelected{
+    border-bottom: 3px solid material.get-color-from-palette($pink-palette, 500) !important;
+}
+p.isSelected{
```

Comment:
Please reformat to the css standard of kebab-case

Reply:
Fixed it.